src/common/views/publication.ts

```diff
@@ -31,6 +38,9 @@ export interface PublicationView extends Identifiable {
     cover?: CoverView;
     customCover?: CustomCoverView;
 
+    duration: ITimeDuration;
+    nbOfTracks: number;
```

Comment:
These fields should be optional, I think.
(I mean: question mark `?` before the colon character `:`)

Reply:
https://github.com/readium/readium-desktop/pull/973/commits/b38967b14f3898ee4788b5a41a2392815fe9294e